pkgs/top-level/perl-packages.nix

```
@@ -10757,6 +10759,10 @@ let
       sha256 = "0crlxmaa4lsgdjm5p9ib8rdxiy70qj1s68za3q3v57v8ll6s4hfx";
     };
     propagatedBuildInputs = [ commonsense ];
+
+    meta = with stdenv.lib; {
+      platforms = platforms.linux;
+    };
```

Since this adds a new `meta` attr, maybe the subAttrs for `description` and `license` can be added ([taken from metacpan](https://metacpan.org/pod/release/MLEHMANN/Linux-Inotify2-2.2/Inotify2.pm)?)
```suggestion
    meta = {
      description = "Scalable directory/file change notification for Perl on Linux";
      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
      platforms = stdenv.lib.platforms.linux;
    };
```

Though I suppose this might be better off in a separate PR altogether.

---

This is a backport of a fix. New metadata should be added in master and doesn't need backport imo.

---

Yep, I'll go ahead and make that PR to master, thanks :muscle: